src/evm/EvmStateMachine.ts

```diff
@@ -90,12 +90,27 @@ class EvmStateMachine extends AStateMachine {
 
         try {
             const result = await this.contractExecuter.executeCall(encodedData);
+            // Decode the return values: (bool success, ExitChannel[] exitChannels)
+            const hexResult = ethers.hexlify(result.returnValue);
+            const [success, exitChannels] =
+                ethers.AbiCoder.defaultAbiCoder().decode(
+                    [
+                        "bool",
+                        "tuple(address participant, bool isPartialExit, uint256 amount, bytes data)[]"
```

Comment:
Use the ethers type defined in @types/ethers so if it changes it only needs to change in the single location